src/assets/translations/de/main.json

```diff
@@ -2488,7 +2488,7 @@
     "foxPowerDiscount": "FOX Power Rabatt",
     "basedOnFee": "Basierend auf einer Gebühr von: %{fee}",
     "feeSummary": "Gebührenübersicht",
-    "simulateFee": "Rabatt simulieren",
+    "simulateFee": "FOX Rabatt simulieren",
```

Comment:
Is this in this diff on purpose?

Reply:
pretty sure having non-EN will rug @firebomb1

Reply:
Yes, I'd appreciate if we left translations to translators 😅 

If you want to invalidate/delete translations because they pose a risk to the users (outdated/wrong info), feel free to do it (or ping me to do it). If the translation can remain outdated until the next translation sync without harm, you can leave them untouched and I'll invalidate them before presenting the strings to translators (so they will re-translate them).

TL;DR: In this case you should likely not modify the translations and remove these changes.

Reply:
Yeah it was on purpose, wanted to update all the translations already!

Reply:
Reverted in [d39f26b](https://github.com/shapeshift/web/pull/8896/commits/d39f26b0e8d2ca219ef67c7d0499745c906e02bd)